components/picker/stories/picker.stories.js

```diff
@@ -97,7 +97,7 @@ export default {
 		},
 		isHovered,
 		isActive,
-		content: { table: { disable: true } },
+		popoverContent: { table: { disable: true } },
```

Comment:
During the picker migration, we tried to clarify this picker control by calling it `popoverContent`. This is the content that gets passed to the popover component in the template. 

Popover has its own `content` arg, so that's why you'll see the `content: popoverContent` assignment.

Reply:
Separately, there might be more that we can pull from the docs migration PR listed above. It looks like some of those additional controls/args (like `currentValue`, `isHovered`, `showWorkflowIcon`, and maybe some others) are missing currently. That could probably be figured out in a separate PR.